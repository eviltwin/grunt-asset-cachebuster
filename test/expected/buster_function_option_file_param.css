h1 {
  background-image: url(hello.png?v=squirrel);
}

h2 {
  background-image: url(//cdn.1.test.com/hello.png?v=squirrel);
}

h3 {
  background-image: url(//cdn.2.test.com/hello.png?v=squirrel);
}

h4 {
  background-image: url(data:image/png;base64,TEST==);
}

h5 {
  background-image: url('data:image/png;base64,TEST==');
}
