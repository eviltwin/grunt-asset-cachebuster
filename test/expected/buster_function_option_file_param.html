<!doctype html>
<html>
  <head>
    <title>test</title>
    <script src="test.js?v=fish"></script>
    <script>
      console.log('testing');
    </script>
    <link href="test.css?v=fish" rel="stylesheet">
    <link href="//cdn.1.test.com/test.css?v=fish" rel="stylesheet">
  </head>
  <body>
    <img src="testing.png?v=fish" alt="">
    <img src="testing.gif?v=fish" alt="">
    <img src="testing.jpg?v=fish" alt="">
    <img src="testing.jpeg?v=fish" alt="">
    <script src="test2.js?v=fish"></script>
    <script src="//cdn.2.test.com/test2.js?v=fish"></script>
  </body>
</html>
